src/main/java/com/EstimatedDeliveryDate/Entity/Holiday.java

```diff
@@ -0,0 +1,41 @@
+package com.EstimatedDeliveryDate.Entity;
+
+import java.io.Serializable;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBDocument;
+
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/*Document object for the Holiday List*/
+
+@Setter
+@NoArgsConstructor
+@DynamoDBDocument
+public class Holiday  implements Serializable {
+	private static final long serialVersionUID = 1L;
+	private String holidayDate;
+	private Integer deliveryHoliday;
+	private Integer pickUpHoliday;
+	private Integer processingHoliday;
```

Comment:
The logical meaning for these values is true or false, should we take them as boolean?

Reply:
Can we keep it as it is for now as I have data in dynamodb table as number